config/opensearch_dashboards.yml

```diff
@@ -107,6 +107,9 @@
 # Logs queries sent to OpenSearch. Requires logging.verbose set to true.
 #opensearch.logQueries: false
 
+# Disables errors from client and enables you to utilize protected words within cluster
```

Comment:
could we add a little information to share this config is passing to initialize opensearch-js

Reply:
Do we need any sort of warning about what the gotchas are of disabling this? I assume there's a good reason for why we don't want to enable this by default.

Reply:
Addressed @seraphjiang @joshuarrrr